legislation_analysis/utils/constants.py

```diff
@@ -29,9 +30,21 @@
 )
 CONGRESS_DATA_CLUSTERED_FILE_NAME = "congress_legislation_clustered.fea"
 CONGRESS_DATA_POS_TAGGED_FILE_NAME = "congress_legislation_pos.fea"
-CONGRESS_DATA_POS_TAGGED_FILE = os.path.join(
+CONGRESS_TOPIC_MODEL = "congress_tm.model"
+CONGRESS_DYN_TOPIC_MODEL = "congress_dyn_tm.model"
```

Comment:
Would you be cool with `FILE_NAME` to the constant so that it's consistent with the rest?